ui/components/app/confirm/info/row/row.tsx

```diff
@@ -106,7 +106,7 @@ export const ConfirmInfoRow: React.FC<ConfirmInfoRowProps> = ({
         display={Display.Flex}
         flexDirection={FlexDirection.Row}
         justifyContent={JustifyContent.center}
-        alignItems={AlignItems.center}
+        alignItems={AlignItems.flexStart}
```

Comment:
We need to ensure that it looks correct for all other cases.

Reply:
Is there any special one that requires centralised label that you know?